src/status_im/contexts/communities/sharing/events.cljs

```diff
@@ -38,6 +38,13 @@
                                     :url     %}])]
      {:fx [[:dispatch [:communities/get-community-channel-share-data chat-id on-success]]]})))
 
+(rf/reg-event-fx :communities/share-community-url-qr-code
```

Comment:
this looks strange, why you need this extra event, why can't you dispatch communities/get-community-share-data directly ?

Reply:
I wanted to be consistent with what already was there, Fixed.